technical-reports/color/color-type.md

```diff
@@ -1,22 +1,257 @@
+  <tr>
+    <td>
+      <ul>
+        <li>It is easy to understand compare to other formats</li>
+        <li>Easy to predict value changes</li>
+      </ul>
+    </td>
+    <td>
+      <ul>
+        <li>No supported in older browsers</li>
+      </ul>
+    </td>
+  </tr>
+</table>
+
+<aside class="example">
+
+```json
+{
+  "Majestic magenta": {
+    "h": 300,
+    "s": "100%",
+    "l": "50%",
+    "a": "100%",
+    "type": "color"
+  },
+  "Simple sage": {
+    "h": 152,
+    "s": "22%",
+    "l": "73%",
+    "a": "100%",
+    "type": "color"
+  }
+}
+```
+
+</aside>
+
+Then, the output variables may look like:
+
+<aside class="example">
+
+```scss
+// colors-rgba.scss
+$majestic-magenta: hsl(300, 100%, 50%, 1);
+$simple-sage: hsl(152, 2
+```
+
+</aside>
+
+### Hex8
+
+Hex8 uses two extra digits, known as the alpha value, to change the transparency of the color. The format follows `#RRGGBBAA`. [Learn more about alpha values in hex.](https://www.digitalocean.com/community/tutorials/css-hex-code-colors-alpha-values#adding-an-alpha-value-to-css-hex-codes)
+
+<table>
+  <tr>
+    <td><strong>Pros</strong></td>
+    <td><strong>Cons</strong></td>
+  </tr>
+  <tr>
+    <td>
+      <ul>
+        <li>Can define alpha value with color</li>
+      </ul>
+    </td>
+    <td>
+      <ul>
+        <li>Less commonly used</li>
+        <li>Alpha value is not immediately obvious (needs calculation)</li>
+        <li>
+          Not available in older versions of internet explorer (<a
+            href="https://caniuse.com/css-rrggbbaa"
+            >caniuse reference</a
+          >)
+        </li>
+      </ul>
+    </td>
+  </tr>
+</table>
 
 <aside class="example">
 
 ```json
 {
   "Majestic magenta": {
-    "$value": "#ff00ff",
-    "$type": "color"
+    "value": "#ff00ff80",
+    "type": "color"
   },
-  "Translucent shadow": {
-    "$value": "#00000088",
-    "$type": "color"
+  "Simple sage": {
+    "value": "#abcabc80",
+    "type": "color"
   }
 }
 ```
 
 </aside>
+
+Then, the output variables may look like:
+
+<aside class="example">
+
+```scss
+// colors-hex.scss
+$majestic-magenta: #ff00ff80;
+$simple-sage: #abcabc80;
+
+// colors-rgba.scss
+$majestic-magenta: rgba(255, 0, 255, 0.5);
+$simple-sage: rgba(171, 202, 188, 0.5);
+```
+
+</aside>
+
+### LCH (Lightness Chroma Hue)
+
+Formatted in L (lightness), C (chroma), H (hue) and an optional (A) alpha. Hue values range from 0 to 360, saturation and lightness are percentage values that go from 0% to 100%, and alpha value can range from 0 and 1 (i.e 0.5) or a percentage (i.e 50%) where 1 or %100 is full opacity (which is the default value if a value isn’t provided).
+
+<table>
+  <tr>
+    <td><strong>Pros</strong></td>
+    <td><strong>Cons</strong></td>
+  </tr>
+  <tr>
+    <td>
+      <ul>
+        <li>Access to 50% more colors (P3 color space)</li>
+        <li>Colors more perceptually uniform</li>
+      </ul>
+    </td>
+    <td>
+      <ul>
+        <li>No fully supported (only safari)</li>
```

Comment:
```suggestion
        <li>Not fully supported (only safari)</li>
```

Reply:
Just a nitpick. It would be nice if the Con sections mentioning the lack of browser support were consistent.

Something like 

> "Limited Browser Support (Safari)"

Consistent verbiage and browsers with support are listed.